src/main/java/com/minelittlepony/hdskins/client/ProfileRepository.java

```diff
@@ -16,16 +17,27 @@
 import net.minecraft.client.texture.AbstractTexture;
 import net.minecraft.client.util.DefaultSkinHelper;
 import net.minecraft.util.Identifier;
+import org.apache.logging.log4j.LogManager;
 
+import java.net.URI;
+import java.net.URISyntaxException;
 import java.nio.file.Path;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.TimeUnit;
 
 public class ProfileRepository {
 
+    private static Set<String> LOCALHOST = ImmutableSet.of(
+            "localhost",
```

Comment:
`lvh.me`

Reply:
If you want that, add it to the whitelist in the config file.

Reply:
That is one of the aliases for `localhost`.

Reply:
Doesn't look like it to me.
```
$ cat /etc/hosts
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
```

Reply:
https://www.quora.com/What-is-lvh-me

Reply:
That's not hard-coded on the client. It comes from dns, so you can change it in dns. I'm not hardcoding something whose resolution isn't also hardcoded.

Reply:
Alright, fair enough.